services/ui/src/components/Accordion/index.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, Fragment } from "react";
```

Comment:
So - I want to make sure of this, are these new components you're adding to the root namespace generic enough to _be_ in the root namespace? The last accordion I moved as a sub-element of the problems space because there were specific assumptions made about its use.

Reply:
After another think about this - I've pull out the component specific stuff for this core accordion and moved all the logic that modifies the columns into the parent components. The only think left is some css that needs to be set for define column widths, but I don't think that is a problem?